openwisp_firmware_upgrader/hardware.py

```diff
@@ -39,6 +39,84 @@
                 'ar71xx-generic-ubnt-airrouter-squashfs-sysupgrade.bin',
                 {'label': 'Ubiquiti AirRouter', 'boards': ('Ubiquiti AirRouter',)},
             ),
+            (
+                'zbt-wg3526-16M-squashfs-sysupgrade.bin',
```

Comment:
looking at https://openwrt.org/toh/hwdata/zbt/zbt_wg3526_16m, I think this should be `ramips-mt7621-zbtlink_zbt-wg3526-16m-squashfs-sysupgrade.bin`

this key should be the image file name without `openwrt-`.

Reply:
ramips-mt7621-zbt-wg3526-16M-squashfs-sysupgrade.bin